### ISSUE TITLE
Pressing 'return' leaves a newline after a message sends

### ISSUE THREAD

**Issue by smyrick** [sindresorhus/caprine#25]
### Description
As expected, if the text input box is empty the placeholder text appears "Type a message...".

On typing a message ("test 1 2 3"), then pressing return to send, the following happens:

* The message "test 1 2 3" sends and appears as expected in the conversation area.
* The input box now has a new line in it, so it is two lines tall, which is not expected. This also makes the placeholder text disappear since the box is no longer empty.

### User Path
<img width="408" alt="screen shot 2015-10-23 at 4 22 39 pm" src="https://cloud.githubusercontent.com/assets/2446877/10707119/c61144e0-79a2-11e5-9f31-bf29e768523d.png">
<img width="416" alt="screen shot 2015-10-23 at 4 22 50 pm" src="https://cloud.githubusercontent.com/assets/2446877/10707120/c7bc549c-79a2-11e5-9b4d-28b0db5d21c8.png">
<img width="425" alt="screen shot 2015-10-23 at 4 22 58 pm" src="https://cloud.githubusercontent.com/assets/2446877/10707121/c9a8f7ba-79a2-11e5-94b8-89fda3657e1d.png">

-
Caprine 0.2.0
darwin x64 14.5.0

**Comment by silvenon** [comment]
Yep, happens to me as well.

**Comment by sindresorhus** [comment]
I don't have time to look into this right now. Help welcome. It might be any of the CSS overrides causing it https://github.com/sindresorhus/caprine/blob/master/browser.css.

**Comment by sindresorhus** [comment]
Looked into this and it's not the Caprine CSS override causing it. It's not happening in Chrome, but might be an issue in the Chrome version used by latest Electron.

**Comment by maciekwyka** [comment]
in 0.3.0 it's still happens.

**Comment by dogancelik** [comment]
I can confirm in v0.4.0

**Comment by jamiees2** [comment]
This looks to be an issue with Electron, as it is also present in https://github.com/danieltian/electron-messenger

Some investigation seemed to show that Messenger was forgetting to clear the textbox after sending the message. I have no idea why, possibly it's an events issue? Maybe the Electron team and/or the Messenger team could shed some light on this.

**Comment by yoannmoinet** [comment]
still happening in v0.5.1 on windows.

**Comment by jayxni** [comment]
I fixed this by calling this JS before mainWindow.show() in 'dom-ready':

```
document.querySelector("._5irm").onkeydown = function(e) {
  if (e.keyCode == 13 && !e.shiftKey) {
    e.preventDefault();
  }
}
```

**Comment by phamdaniel** [comment]
Apparently fixed as of v0.6.0

**Comment by silvenon** [comment]
Yep, it's fixed.
